x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_summary.ts

```diff
@@ -51,7 +51,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   describe('Dataset quality summary', () => {
     before(async () => {
       await synthtrace.index(getInitialTestLogs({ to, count: 4 }));
-      await PageObjects.svlCommonPage.loginAsViewer();
+      await PageObjects.svlCommonPage.loginAsEditor(); // TODO: Not sure whether this is right
```

Comment:
This should not be necessary, just testing whether it makes the tests pass. By my understanding, the observability viewer should have read access to integrations, but it looks like it doesn't.